src/libcharon/plugins/vici/python/vici/session.py

```diff
@@ -1,12 +1,29 @@
 import socket
 
 from .exception import SessionException, CommandException, EventUnknownException
-from .protocol import Transport, Packet, Message
+from .protocol import Transport, Packet, Message, RECV_TIMEOUT_DEFAULT
 from .command_wrappers import CommandWrappers
 
 
 class Session(CommandWrappers, object):
     def __init__(self, sock=None):
+        """Establish a session with a StrongSwan daemon. By default, the Session
+        will connect to StrongSwan via the `/var/run/charon.vici` Unix Domain
+        Socket.
+
+        If there is a need to connect a socket in another location or set
+        specific settings on the socket like a timeout, create a connection and
+        pass it to the `sock` parameter.
+
+        .. note::
+
+            In case a timeout is set on the socket, the internal read code
+            will temporarily disable it after receiving the first byte to avoid
+            partial read corruptions.
+
+        :param sock: Socket connected to StrongSwan (optional)
```

Comment:
```suggestion
        """Establish a session with an IKE daemon.

        By default, the session will connect to the `/var/run/charon.vici` Unix
        domain socket.

        If there is a need to connect a socket in another location or set
        specific settings on the socket (like a timeout), create and connect
        a socket and pass it to the `sock` parameter.

        .. note::

            In case a timeout is set on the socket, the internal read code
            will temporarily disable it after receiving the first byte to avoid
            partial read corruptions.

        :param sock: socket connected to the IKE daemon (optional)
```

Reply:
Applied, thanks.